pkg/cloud/ratelimit_test.go

```diff
@@ -98,8 +98,9 @@ func TestTickerRateLimiter(t *testing.T) {
 	if elapsed > time.Second {
 		t.Errorf("TickerRateLimiter.Accept took too long: %v, want <1s", elapsed)
 	}
-	if elapsed < 500*time.Millisecond {
-		t.Errorf("TickerRateLimiter.Accept took too short: %v, want >500ms", elapsed)
+	// First request returns immediately.
```

Comment:
First request will be called immediately without delay not returned. Can you rephrase this?